modules/php/Actions/WriteX.php

```diff
@@ -34,6 +35,8 @@ public function actWriteX(string $slotId)
     $number = NUMBER_X;
     $player = $this->getPlayer();
     $scribble = $player->scoresheet()->addScribble($slotId, $number);
+    Stats::incNumberedSpacesNumber($player->getId(), 1);
+    Stats::setEmptySlotsNumber($player->getId(), $player->scoresheet()->countAllUnscribbledSlots());
```

Comment:
Looks like the name of the stat is not consistent with the functions that counts all unscribbled slots, and not just unscribbled number slots

Reply:
See other related comment